source/daplink/usb2uart/usbd_user_cdc_acm.c

```diff
@@ -85,7 +85,11 @@ int32_t USBD_CDC_ACM_PortSetLineCoding(CDC_LINE_CODING *line_coding)
     UART_Config.DataBits    = (UART_DataBits) line_coding->bDataBits;
     UART_Config.Parity      = (UART_Parity)   line_coding->bParityType;
     UART_Config.StopBits    = (UART_StopBits) line_coding->bCharFormat;
+#if defined(BOARD_VBLUNO51) || defined(BOARD_VBLUNO52)
+    UART_Config.FlowControl = UART_FLOW_CONTROL_RTS_CTS;
+#else
     UART_Config.FlowControl = UART_FLOW_CONTROL_NONE;
+#endif    
```

Comment:
@c1728p9 The VBLUno51, VBLUno52 boards `always use` UART Hardware flow control (RTS/CTS) feature for DAPLink interface